lib/chef/provider/user/linux.rb

```diff
@@ -29,14 +29,16 @@ def load_current_resource
         end
 
         def compare_user
-          super
+          user_changed = super
```

Comment:
May worth adding a `@change_desc ||= []` here if we assume a base class change shouldn't break this class behavior.